test/TestConfig/scripts/testKitGen/resultsSummary/resultsSum.pl

```diff
@@ -30,6 +30,26 @@
 my $failuremkarg;
 my $tapFile;
 my $diagnostic = 'failure';
+my $jdkVersion = "";
+my $jdkImpl = "";
+my $buildList = "";
+my $spec = "";
+my %spec2jenkinsFile = (
+	'linux_x86-64_cmprssptrs'    => 'openjdk_x86-64_linux',
+	'linux_x86-64'               => 'openjdk_x86-64_linux_xl',
+	'linux_arm'                  => 'openjdk_aarch32_linux',
+	'linux_aarch64_cmprssptrs'   => 'openjdk_aarch64_linux',
+	'linux_ppc-64_cmprssptrs'    => 'openjdk_ppc64_aix',
+	'linux_ppc-64_cmprssptrs_le' => 'openjdk_ppc64le_linux',
+	'linux_390-64_cmprssptrs'    => 'openjdk_s390x_linux',
+	'zos_390-64_cmprssptrs'      => 'openjdk_s390x_zos',
+	'osx_x86-64_cmprssptrs'      => 'openjdk_x86-64_mac',
+	'osx_x86-64'                 => 'openjdk_x86-64_mac_xl',
+	'win_x86-64_cmprssptrs'      => 'openjdk_x86-64_windows',
+	'win_x86-64'                 => 'openjdk_x86-64_windows_xl',
+	'win_x86'                    => 'openjdk_x86-32_windows',
+	'osx_sparcv9-64'             => 'openjdk_sparc_solaris',
```

Comment:
`osx_sparcv9-64` will be changed soon https://github.com/AdoptOpenJDK/openjdk-tests/pull/1319

Reply:
changed based on the new name